core/src/worker/workflow/machines/child_workflow_state_machine.rs

```diff
@@ -449,6 +450,15 @@ impl ChildWorkflowMachine {
                 cancelled_before_sent: false,
             },
         );
+        let user_metadata = if attribs.static_summary.is_some() || attribs.static_details.is_some()
+        {
+            Some(UserMetadata {
+                summary: attribs.static_summary.clone(),
+                details: attribs.static_details.clone(),
```

Comment:
This is a little nitpicky, but, instead of cloning here, since these values aren't used by `start_child_workflow_cmd_to_api`, let's instead https://doc.rust-lang.org/std/mem/fn.take.html the string out and we avoid a clone